lib/generators/active_admin/install/templates/active_admin.rb.erb

```diff
@@ -163,7 +163,10 @@ ActiveAdmin.setup do |config|
   #
   # Set the localize format to display dates and times.
   # To understand how to localize your app with I18n, read more at
-  # https://github.com/svenfuchs/i18n/blob/master/lib%2Fi18n%2Fbackend%2Fbase.rb#L52
+  # https://guides.rubyonrails.org/i18n.html
+  #
+  # You can run bin/rails runner 'puts I18n.t("date.formats")' to see the
```

Comment:
```suggestion
  # You can run `bin/rails runner 'puts I18n.t("date.formats")'` to see the
```

Reply:
I thought about doing it, but I did not know if you would accept it :joy:

Reply:
Yeah, I included the backquotes in my comment by I forgot to skip the backquotes, so they didn't appear and instead formatted the command :sweat_smile: